addons/backup-and-maintenance.sh

```diff
@@ -113,7 +109,7 @@ should_backup(){
 }
 
 backup_db(){
-    /usr/local/pf/addons/database-cleaner.pl --table=locationlog_archive --date-field=end_time --older-than="1 MONTH"
+    /usr/local/pf/addons/database-cleaner.pl --table=locationlog_history --date-field=end_time --older-than="1 MONTH"
```

Comment:
This is already cleaned by the locationlog_cleanup job so there is no point on even keeping that line

Reply:
Done